src/metabase/query_processor/util/add_alias_info.clj

```diff
@@ -117,7 +117,9 @@
       [:expression expression-name])
 
     [:aggregation index opts]
-    (if-let [opts (remove-namespaced-options opts)]
+    (if-let [opts (-> opts
```

Comment:
@metamben why this is needed? my understanding was that this is a lightweight change that should have no effect on the existing logic

Reply:
As this is a public function in a util namespace, I haven't really studied where it makes a difference. I wouldn't be surprised if in tests only. I'll have a closer look.

Reply:
Actually, the thing is that now the aggregation references in the `order-by` clause keep `base-type`.